package.json

```diff
@@ -32,13 +34,15 @@
     "valine": "^1.5.1"
   },
   "devDependencies": {
-    "@astrojs/ts-plugin": "^1.1.3",
+    "@astrojs/ts-plugin": "^1.3.1",
+    "@biomejs/biome": "1.4.1",
     "@iconify-json/fa6-brands": "^1.1.13",
     "@iconify-json/fa6-regular": "^1.1.13",
     "@iconify-json/fa6-solid": "^1.1.15",
     "@iconify-json/material-symbols": "^1.1.59",
     "@rollup/plugin-yaml": "^4.1.2",
     "@tailwindcss/typography": "^0.5.10",
+    "astro-compress": "github:astro-community/AstroCompress#no-sharp",
```

Comment:
maybe fixed  
I have temporarily changed to the no-sharp branch.  
I've already heard that there are different versions of Sharp, and I was wondering whether to fix the version to Astro or use no-sharp, but I'm currently using no-sharp.